dbcon/dmlpackage/vendordmlstatement.cpp

```diff
@@ -25,6 +25,8 @@
 #include "vendordmlstatement.h"
 #undef VENDORDMLSTATEMENT_DLLEXPORT
 
+#include "exceptclasses.h"
```

Comment:
It looks like this compilation unit doesn't need this include. What do you think?